### PR TITLE
Worker docs cleanup for 0.13

### PR DESCRIPTION
**Pull request by xingluw**
Cleaning up some text and formatting for 0.13, in the future we will restructure the worker docs so that it is easier to understand and use, as well as accommodating for future changes.

### CODE REVIEW

File: website/content/docs/configuration/worker/kms-worker.mdx [hunk 1]
```diff
@@ -6,7 +6,7 @@ description: |-
 ---
 
 
-## KMS Worker Configuration
+# KMS Worker Configuration
```
Comment (Dan-Heath) [inline website/content/docs/configuration/worker/kms-worker.mdx:9]:
```suggestion
# KMS worker configuration
```

File: website/content/docs/configuration/worker/kms-worker.mdx [hunk 2]
```diff
@@ -87,7 +87,7 @@ key; in production you'd want to use a KMS such as AWS KMS, GCP CKMS, Azure
 KeyVault, or HashiCorp Vault. For a complete guide to all available KMS types,
 see our [KMS documentation](/boundary/docs/configuration/kms).
 
-# Complete Configuration Example
+## Complete Configuration Example
```
Comment (Dan-Heath) [inline website/content/docs/configuration/worker/kms-worker.mdx:90]:
```suggestion
## Complete configuration example
```

**Review by Dan-Heath — APPROVED** [review]
I added suggestions for a couple headings but otherwise LGTM. Thank you for creating this!

File: website/content/docs/configuration/worker/pki-worker.mdx [hunk 3]
```diff
@@ -74,6 +74,6 @@ kms "aead" {
 ```
 
 ~> **Note:** `name` and `description` fields are not valid config fields for PKI
-workers. These fields are only valid for [KMS Workers][]. `name` and
+workers. These fields are only valid for [KMS Workers](/boundary/docs/configuration/worker/kms-worker). `name` and
```
Comment (xingluw) [inline website/content/docs/configuration/worker/pki-worker.mdx:77]:
```suggestion
workers. These fields are only valid for [KMS Workers][]. `name` and
```

File: website/content/docs/configuration/worker/pki-worker.mdx [hunk 4]
```diff
@@ -74,6 +74,6 @@ kms "aead" {
 ```
 
 ~> **Note:** `name` and `description` fields are not valid config fields for PKI
-workers. These fields are only valid for [KMS Workers][]. `name` and
+workers. These fields are only valid for [KMS Workers]](/boundary/docs/configuration/worker/kms-worker). `name` and
```
Comment (xingluw) [inline website/content/docs/configuration/worker/pki-worker.mdx:77]:
```suggestion
workers. These fields are only valid for [KMS Workers](/boundary/docs/configuration/worker/kms-worker). `name` and
```

**Pull request merged**